src/meadowdb/connection.py

```diff
@@ -94,8 +96,8 @@ def set_default_userspace_spec(
 @dataclasses.dataclass(frozen=True)
 class MeadowdbEffects:
     """
-    This should be part of meadowflow.effects, but we want meadowflow to depend on
-    meadowdb, not the other way around.
+    This should be part of meadowflow.effects, but we want meadowflow to depend
```

Comment:
I think you might have your comment reformatter set to 80 characters? I think this comment was fine before the change? (Under 88 characters)

Reply:
Yeah, took me a while initially to configure everything to 88, should be good now.